### DISCUSSION TITLE
Update on updates to howler

### DISCUSSION THREAD

**Discussion by goldfire** [goldfire/howler.js#1594] [Announcements]
Hey everyone! First, thanks so much for all of the contributions to the library and to these discussions!

I mainly just wanted to apologize for my radio silence on here the last few months. My game studio is working on a new game that has been all-consuming lately as we prepare for our first public demo next week (followed by E3-adjacent showcases a month later).

I just wanted to give a quick update that I've still been keeping track of issues/pull requests and plan to get back to a more regular pace of updates once I get through this stretch.

Thanks for the understanding, and I look forward to continuing to see all of the great audio experiences everyone keeps making for the web!

**Comment by rogihee** [comment]
Thanks for making this awesome lib and thank you for the update!

**Comment by amcjen** [comment]
Yess, appreciate this update! Was just scouring through pull requests/issues to see if the project was still viable. So glad to hear it still is!

@goldfire what would help you the most in getting some of the really desirable pull-requests merged in? (For me, it'd be the ES6 one and the iOS control center one.)  

If you need testing done, or donations or whatever, let us know! I'm sure I speak for a lot of us in that we really appreciate this library and would love to see it continue. 🥰

**Comment by lustremedia** [comment]
Hi James @goldfire what is the state of howler.js are there any updates planned soon? Thanks for all your hard work.

**Comment by tobias-graf-p** [comment]
Hi @goldfire 
Can you give another update on your current situation and plans regarding this library?

**Comment by LetMyPeopleCode** [comment]
Hi @goldfire 
Echoing @tobias-graf-p ... Is this still being actively maintained? I see you updated the backers file 4 months ago, but the last pull request was merged over a year ago. 

Curious as Howler is a component of a project I'm working on and I'm considering some updates to it in my roadmap. Wondering if it would be good to upstream them as PRs or just maintain my own fork.

**Comment by catFurr** [comment]
hi @goldfire any update on this topic?

**Comment by mgussekloo** [comment]
any update on this topic?

**Comment by ManiMatter** [comment]
*bump* - same question here. Is this still actively maintained? The PR history looks a bit outdated. If not - would you have any pointers on alternatives you can recommend?

**Comment by NoodleSushi** [comment]
any update on this topic?
